### DISCUSSION TITLE
Wrong gender - build

### DISCUSSION THREAD

**Discussion by Frankiesomeone** [glucauze/sd-webui-faceswaplab#97] [General]
I'm finding that with female faces that are not super feminine, the face checkpoint builder often gets the gender wrong. It would be neat to be able to select it manually.

**Comment by glucauze** [comment]
Yes, i will think of it.

**Comment by glucauze** [comment]
I have implemented it in v1.2.7 in pull request. You can test it if you want.

**Reply by Frankiesomeone** [reply]
fantastic, just tried it and it works like a charm
